Blog/gst-build/cross-building-with-gst-build.md

```diff
@@ -3,27 +3,27 @@
 ### A brief Introduction
 
 gst-build is one of the two build system used by the community to hack into the whole GStreamer solution.
-A previous blogpost here has been released to present gst-build and how to hack into it.
+A previous blogpost [here](https://www.collabora.com/FIXME) has been released to present gst-build and how to hack into it.
```

Comment:
we can change it to https://www.collabora.com/news-and-blog/ in case of using it without chagin it ;)